Assets/MRTK/Core/Providers/InputSimulation/InputSimulationEnum.cs

```diff
@@ -43,25 +44,58 @@ public enum EyeGazeSimulationMode
     }
 
     /// <summary>
-    /// Defines for how input simulation handles hands
+    /// Defines for how input simulation handles controllers
     /// </summary>
-    public enum HandSimulationMode
+    public enum ControllerSimulationMode
     {
 
         /// <summary>
-        /// Disable hand simulation
+        /// Disable controller simulation
         /// </summary>
         Disabled,
 
         /// <summary>
-        /// Raises gesture events only
+        /// Raises hand gesture events only
         /// </summary>
-        Gestures,
+        HandGestures,
 
         /// <summary>
         /// Provide a fully articulated hand controller
         /// </summary>
-        Articulated,
+        ArticulatedHand,
+
+        /// <summary>
+        /// Provide a 6DoF motion controller
+        /// </summary>
+        MotionController,
+    }
+
+    /// <summary>
+    /// Defines for how input simulation handles controllers
+    /// </summary>
+    [Obsolete("Use ControllerSimulationMode instead.")]
+    public enum HandSimulationMode
+    {
```

Comment:
nit: add a region Obsolete Properties for consistency with the other files

Reply:
Fixed!